### DISCUSSION TITLE
마인드맵

### DISCUSSION THREAD

**Discussion by TRENDLMC** [TRENDLMC/shoppingmall#1] [Announcements]

**Comment by TRENDLMC** [comment]
![image](https://user-images.githubusercontent.com/130523481/236112951-9b7d6a3e-b053-4eaf-b32d-b47cf49468dd.png)

**Comment by meanlightk** [comment]
![형상관리](https://user-images.githubusercontent.com/119726468/236115700-a54fab87-30c4-4873-a68a-b89d460f1826.png)

**Comment by Seonz** [comment]
![11](https://user-images.githubusercontent.com/130538564/236118577-41d92f83-0a72-4f20-b664-5593359465b7.png)

**Comment by kimyegun** [comment]
![제목 없음](https://user-images.githubusercontent.com/130538532/236116138-93df830d-b4d2-4f91-8a76-803ed731a56d.png)

**Comment by audelahd** [comment]
![0504](https://user-images.githubusercontent.com/114639084/236125093-96284013-3bfa-4a82-8606-edd3f0e26fdb.PNG)
